packages/@expo/cli/src/export/createMetadataJson.ts

```diff
@@ -38,10 +38,11 @@ export function createMetadataJson({
             bundle: path.join('bundles', fileNames[platform]!),
             // Collect all of the assets and convert them to the serial format.
             assets: bundle.assets
+              .filter((asset: any) => !asset.embedded)
```

Comment:
```suggestion
              .filter((asset) => !asset.embedded)
```

(and add `embedded?: true` to the types)